powertools-serialization/src/main/java/software/amazon/lambda/powertools/utilities/jmespath/Base64GZipFunction.java

```diff
@@ -50,21 +50,21 @@ protected <T> T callFunction(Adapter<T> runtime, List<FunctionArgument<T>> argum
     }
 
     public static String decompress(byte[] compressed) {
-        if ((compressed == null) || (compressed.length == 0)) {
+        if (compressed == null || compressed.length == 0) {
             return "";
```

Comment:
see my other [comment](https://github.com/aws-powertools/powertools-lambda-java/pull/1264/files#r1254293580) on that one